src/frontend/src/lib/stores/transactions.store.ts

```diff
@@ -64,6 +65,7 @@ export const initTransactionsStore = <
 				[tokenId]: null
 			})),
 		reset,
+		resetAll: () => update(() => ({})),
```

Comment:
We generally use null for reset because it explicitly indicates a decision to reset, such as when an error occurs. However, in this case, you are setting to the initial value, which creates a slight discrepancy.

Reply:
I was trying to replicate what is happening for the ETH transactions store too. But for sure I can set `null` on this, and then, in another PR, I adjust that store too

Reply:
I would say so, makes sense?

Reply:
just to clarify, that means setting `null` in all keys, like `{tokenId1: null, tokenId2: null, ...}`, and NOT that the entire store is `null` instead of `{}`

correct?

Reply:
what does not reset?